quic/s2n-quic-platform/src/io/xdp.rs

```diff
@@ -38,6 +38,54 @@ pub mod socket {
     }
 }
 
+pub mod tx {
+    pub use s2n_quic_core::io::tx::*;
+
+    pub fn channel(
+        socket: ::std::net::UdpSocket,
+    ) -> (
+        impl Tx<PathHandle = crate::message::default::Handle>,
+        impl core::future::Future<Output = ::std::io::Result<()>>,
+    ) {
+        // Initial packets don't need to be bigger than the minimum
+        let max_mtu = s2n_quic_core::path::MaxMtu::MIN;
+
+        // It's unlikely the initial packets will utilize, so just disable it
+        let gso = crate::features::Gso::default();
+        gso.disable();
+
+        // compute the payload size for each message from the number of GSO segments we can
+        // fill
```

Comment:
this comment is a little confusing after the previous one about not utilizing gso

Reply:
Yeah that was copy/pasted from the tokio code :smile: